vendors/nxp/LPC54018/bootloader/spifi_boot.c

```diff
@@ -0,0 +1,394 @@
+/*
+ * Copyright 2018 NXP
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * o Redistributions of source code must retain the above copyright notice, this list
+ *   of conditions and the following disclaimer.
+ *
+ * o Redistributions in binary form must reproduce the above copyright notice, this
```

Comment:
This means that even tiny self-contained products have to include some sort of written notice if they use this bootloader. I thought we were trying to avoid such restrictions.